src/main/java/net/atos/zac/formulieren/FormulierRuntimeService.java

```diff
@@ -0,0 +1,272 @@
+            case "ZAAK:GROEP" -> getGroepForZaakDefaultValue(context.getZaak());
+            case "ZAAK:BEHANDELAAR" -> getBehandelaarForZaakDefaultValue(context.getZaak());
+            default -> defaultValue.startsWith(":") ?
+                    context.getZaakData().getOrDefault(defaultValue.substring(1), "").toString() :
+                    defaultValue;
+        };
+    }
+
+    private String getGroepForZaakDefaultValue(final Zaak zaak) {
+        return zgwApiService.findGroepForZaak(zaak)
+                .map(groep -> identityService.readGroup(groep.getBetrokkeneIdentificatie().getIdentificatie()).getName())
+                .orElse(null);
+    }
+
+    private String getBehandelaarForZaakDefaultValue(final Zaak zaak) {
+        return zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)
+                .map(behandelaar -> identityService.readUser(behandelaar.getIdentificatienummer()).getFullName())
+                .orElse(null);
+    }
+
+    private String formatDefaultValue(final String defaultWaarde, final FormulierVeldtype veldtype) {
+        return switch (veldtype) {
+            case CHECKBOX -> formatCheckboxDefaultValue(defaultWaarde);
+            case DATUM -> formatDatumDefaultValue(defaultWaarde);
+            default -> defaultWaarde;
+        };
+    }
+
+    private String formatCheckboxDefaultValue(final String defaultWaarde) {
+        return (StringUtils.equalsIgnoreCase("ja", defaultWaarde) ||
+                StringUtils.equalsIgnoreCase("true", defaultWaarde) ||
+                StringUtils.equals("1", defaultWaarde)) ? BooleanUtils.TRUE : BooleanUtils.FALSE;
+    }
+
+    private String formatDatumDefaultValue(final String defaultWaarde) {
+        if (defaultWaarde.matches("^[+-]\\d{1,4}$")) {
+            int dagen = Integer.parseInt(substring(defaultWaarde, 1));
+            if (defaultWaarde.startsWith("+")) {
+                return LocalDate.now().plusDays(dagen).format(DATUM_FORMAAT);
+            } else {
+                return LocalDate.now().minusDays(dagen).format(DATUM_FORMAAT);
+            }
+        } else {
+            return defaultWaarde;
+        }
+    }
+
+    private String resolveMultipleChoiceOptions(final String meerkeuzeOpties) {
+        final var referenceTableCode = substringAfter(meerkeuzeOpties, "REF:");
+        if (isNotBlank(referenceTableCode)) {
+            final var referenceTable = referenceTableService.readReferenceTable(referenceTableCode);
+            return referenceTable.getValues()
+                    .stream()
+                    .sorted(Comparator.comparingInt(ReferenceTableValue::getSortOrder))
+                    .map(ReferenceTableValue::getName)
+                    .collect(Collectors.joining(REFERENCE_TABLE_SEPARATOR));
+        } else {
+            return meerkeuzeOpties;
+        }
+    }
+
+    private void versturenDocumenten(final FormulierData formulierData) {
```

Comment:
The word `versturen` is a bit confusing (as is `verzend` by the way). If I look at what is actually happening it is just about updating documents in the ZGW API right? 

```suggestion
    private void updateDocumentsFromFormData(final FormulierData formulierData) {
```

Reply:
Ik heb er van gemaakt markDocumentAsSent(). En de methode daar onder markDocumentAsSigned()

Reply:
good one